helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -191,6 +194,9 @@ starrocksCnSpec:
   serviceAccount: ""
   # add annotations for cn pods. example, if you want to config monitor for datadog, you can config the annotations.
   annotations: {}
+  # If runAsNonRoot is true, the container is run as non-root user.
+  # The userId will be set to 1000, and the groupID will be set to 1000.
+  runAsNonRoot: false
```

Comment:
does this configuration overwrite `fsGroup`? or we won't use `fsGroup` any more?